### PR TITLE
feat: classified squads

### PR DESCRIPTION
**Pull request by Crypta-Eve**
Add the ability to hide a squad member list. A squad marked as classified will only show the member list for moderators and admins.

I got delayed so have the merge commit in here... Not sure if thats ideal or not...

### CODE REVIEW

File: src/resources/views/squads/show.blade.php [hunk 1]
```diff
@@ -217,6 +217,7 @@
     </div>
   @endcan
 
+  @if(!$squad->is_classified or $squad->getIsModeratorAttribute() or auth()->user()->isAdmin())
```
Comment (warlof) [inline src/resources/views/squads/show.blade.php:220]: can you use gate here ? https://github.com/eveseat/web/blob/a1348ae96b14c7e62a3a4ad5f6026383a933b8ba/src/Acl/Policies/SquadPolicy.php#L119 / https://github.com/eveseat/web/blob/a1348ae96b14c7e62a3a4ad5f6026383a933b8ba/src/WebServiceProvider.php#L413 :p

(also, adapt the gate to handle this new classified attribute)

File: src/resources/views/squads/edit.blade.php [hunk 2]
```diff
@@ -50,6 +50,19 @@
...
+                  <label for="squad-classified" class="col-sm-1 col-form-label">Classified?</label>
+                  <div class="col-sm-1">
+                    @if($squad->is_classified)
+                      <input type="checkbox" name="classified" id="squad-classified" class="form-check-input ml-0" checked />
+                    @else
+                      <input type="checkbox" name="classified" id="squad-classified" class="form-check-input ml-0" />
+                    @endif
+                  </div>
+                  <div class="col-sm-10">
+                    <div>If a squad is classified, then only moderators will be able to see the member list</div>
```
Comment (warlof) [inline src/resources/views/squads/edit.blade.php:63]: maybe add "moderators and administrators" here ?

File: src/resources/views/squads/create.blade.php [hunk 3]
```diff
@@ -37,6 +37,15 @@
                     </select>
                   </div>
                 </div>
+                <div class="form-group row">
+                  <label for="squad-classified" class="col-sm-1 col-form-label">Classified?</label>
+                  <div class="col-sm-1">
+                    <input type="checkbox" name="classified" id="squad-classified" class="form-check-input ml-0" />
+                  </div>
+                  <div class="col-sm-10">
+                    <div>If a squad is classified, then only moderators will be able to see the member list</div>
```
Comment (warlof) [inline src/resources/views/squads/create.blade.php:46]: maybe add "moderators and administrators" here ?

**Review by warlof — CHANGES_REQUESTED** [review]
except the gate request, it's only suggestion :)

File: src/resources/views/squads/edit.blade.php (hunk 2 above)
Comment (Crypta-Eve) [reply src/resources/views/squads/edit.blade.php:63]: yeah.... added the admin viewing later... added, will commit shortly

File: src/resources/views/squads/create.blade.php (hunk 3 above)
Comment (Crypta-Eve) [reply src/resources/views/squads/create.blade.php:46]: yeah.... added the admin viewing later... added, will commit shortly

**Comment by warlof** [comment]
perfect, thank you :)

**Pull request merged**